### PR TITLE
Products

### PR DESCRIPTION
**Pull request by ttahm3d**
- Modified Header
- Changed homepage
- Added login page
- added signup page
- Added products listing page
- added wishlist page
- added cart page

Homepage
image

Product List page
image

Login
image

Sign Up
image

Wishlist Page
image

Cart Page
image

### CODE REVIEW

**Review by Hamdrive — APPROVED** [review]
Looks good, great practices followed and can be easily merged!
Good to go!

File: src/App.js [hunk 1]
```diff
@@ -5,20 +5,26 @@ import { Suspense, lazy } from "react";
...
-    <>
+    <div className="main__body">
       <BrowserRouter>
         <Header />
         <Suspense fallback="Loading...">
           <Routes>
             <Route exact path="/" element={<Homepage />} />
+            <Route exact path="/products" element={<ProductList />} />
+            <Route exact path="/signup" element={<SignUp />} />
+            <Route exact path="/login" element={<Login />} />
```
Comment (kjais1720) [inline src/App.js:22]: I think exact is deprecated now-a-days, it is used by default maybe

**Review by kjais1720 — APPROVED** [review]

File: src/App.js (hunk 1 above)
Comment (ttahm3d) [reply src/App.js:22]: I'm a migrating user who has not read v6 docs. Spare me.

File: src/components/Header/Header.jsx [hunk 2]
```diff
@@ -26,20 +31,37 @@ export const Header = () => {
...
+    //   id: 3,
+    //   icon: (
+    //     <>
+    //       <div class="avatar">
+    //         <img src={avatar} class="avatar-round avatar-sm" alt="avatar" />
+    //       </div>
+    //     </>
+    //   ),
+    //   title: "User",
+    // },
```
Comment (Sameony) [inline src/components/Header/Header.jsx:52]: remember to clean up unused code blocks before merging

Comment (Sameony) [inline src/components/Header/Header.jsx:52]: remember to clean up unused code blocks before merging

**Review by Sameony — APPROVED** [review]
Loved the descriptive and concise commit messages. Great code structure.

File: src/components/Header/Header.jsx (hunk 2 above)
Comment (ttahm3d) [reply src/components/Header/Header.jsx:54]: it is not unused if i have commented it.

**Pull request merged**